docs/claw/slack.zh.md

```
@@ -0,0 +1,219 @@
+
+!!! tip "令牌格式"
+    - App-Level Token：以 `xapp-` 开头
+    - Bot User OAuth Token：以 `xoxb-` 开头
+
+    如果你的令牌以 `xoxp-` 开头，说明误复制了 User OAuth Token。
+
+## 步骤 4：邀请机器人到频道
+
+在 Slack 中，将机器人邀请到你希望它响应消息的频道：
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)